src/server/dfly_main.cc

```diff
@@ -84,6 +86,9 @@ ABSL_FLAG(MaxMemoryFlag, maxmemory, MaxMemoryFlag(0),
           "Limit on maximum-memory that is used by the database. "
           "0 - means the program will automatically determine its maximum memory usage. "
           "default: 0");
+ABSL_FLAG(string, cluster_mode, "",
```

Comment:
I think moving this flag to server_family.cc is better. 
Then you can do the parsing inside `ServerFamily` constructor and make `is_emulated_cluster` to be a variable in the class.